ericus123/BRAND-API
# Posts api

**What does this PR do?**
Posts crud operation

**Description of Task to be completed?**
Delete posts
Edit posts
Create posts
Get single posts
Get all posts  
**How should this be manually tested?**
Clone this repo [https://github.com/ericus123/BRAND-API/tree/FT-Posts](url)
Switch your working directory cd BRAND-API
Checkout the branch git checkout FT-Posts
In your terminal run npm start
Use to postman to test endpoints (endpoints can be found in **routes/posts.js**)
You should perform all posts  crude operations successfully

**Any background context you want to provide?**
You must have node js ,mongo db and express js already installed 
**What are the relevant  trello stories?**
https://trello.com/c/J4CnXujX/61-posts-crud-operationsbrand-api
